views/vizor2Teaser.handlebars

```diff
@@ -0,0 +1,91 @@
+                                <g id="Group-2-Copy" transform="translate(16.000000, 16.000000) rotate(-270.000000) translate(-16.000000, -16.000000) ">
+                                    <ellipse id="Oval" stroke-width="2" cx="16" cy="16" rx="15" ry="15"></ellipse>
+                                    <g id="Group" transform="translate(6.857143, 6.857143)" stroke-width="2">
+                                        <g id="Group-2">
+                                            <path d="M0,9 L17.0666667,9" id="Line-Copy" stroke-linecap="square"></path>
+                                            <polyline id="Rectangle-5" transform="translate(9.633145, 9.208875) rotate(-45.000000) translate(-9.633145, -9.208875) " points="16.0331451 2.80887515 16.0331451 15.6088752 3.23314505 15.6088752"></polyline>
+                                        </g>
+                                    </g>
+                                </g>
+                            </g>
+                        </g>
+                    </g>
+                </svg>
+                <span class="scroll-down-button__label">Show more</span>
+            </button>
+        </div>
+    </section>
+
+    <section class="section introduction">
+        <div class="section__wrapper">
+            <p class="kicker">Introducing the Vizor 360 beta</p>
+            <h2 class="h2">Dead simple WebVR.<br>Drag, drop, publish.</h2>
+            <img class="introduction__screenshot" src="/images/vizor2-teaser/editor-screenshot.png" alt="" />
+        </div>
+    </section>
+
+    <section class="section register">
+        <div class="section__wrapper">
+            <p class="kicker">Beta test vizor 360</p>
+            <h2 class="h2">Like a test drive?<br>Register now.</h2>
+            <form action="//vizor.us9.list-manage.com/subscribe/post?u=18aabb3ca7b701a5acacc8aca&amp;id=97a6941be8" method="post" id="mc-embedded-subscribe-form" name="mc-embedded-subscribe-form" class="register__form" target="_blank" novalidate>
+                <div class="register__email-container">
+                	<input type="email" placeholder="Email Address" value="" name="EMAIL" class="register__email text-input" id="mce-EMAIL">
+                    <p class="register__hint">We will not send you any spam, bacon nor any other pork products.</p>
+                </div>
+                <div style="position: absolute; left: -5000px;" aria-hidden="true"><input type="text" name="b_18aabb3ca7b701a5acacc8aca_97a6941be8" tabindex="-1" value=""></div>
+                <button class="register__submit button">Register</button>
+            </div>
+        </div>
+    </section>
+
+    <footer class="footer section">
+        <div class="section__wrapper footer__wrapper">
+            <ul class="footer__link-list">
+                <li class="footer__link-list-item"><a href="https://vizorvr.herokuapp.com/" target="_blank">Public Slack</a></li>
```

Comment:
It is references in the styles: https://github.com/vizorvr/vizor/pull/1994/files#diff-193219a823aec21c2bec8feccf3b5f3eR383